webapp/src/DOMJudgeBundle/Security/DOMJudgeIPAuthenticator.php

```diff
@@ -23,13 +23,16 @@ class DOMJudgeIPAuthenticator extends AbstractGuardAuthenticator
     private $security;
     private $container;
     private $em;
+    private $dj;
```

Comment:
nit: we typically name this variable $DOMJudgeService, we should do so here too for consistency. We could in a later step rename all of this to $dj, which is much shorter and nicer.